backend/internal/service/handler/spotify/handler.go

```diff
@@ -17,8 +19,15 @@ func NewHandler(store storage.Storage, config config.Spotify) *Handler {
 		spotifyauth.WithRedirectURL(config.RedirectURI),
 		spotifyauth.WithScopes(spotifyauth.ScopeUserReadPrivate),
 	)
+	session := session.New(session.Config{
+		Storage:    store,
```

Comment:
any reason for having this function take in store and passing it here? since this function being called in server.go with memory.New and session.New creates memory.New by default

Reply:
maybe to use a store thats already defined, but this store already has configurations specific to session values right?

Reply:
i was being lazy, yes, you should prob drill down the `*session.Store` instead of the `storage.Storage` used within the session